core/misc/tabledrag.js

```diff
@@ -240,7 +241,8 @@ Backdrop.tableDrag.prototype.hideColumns = function () {
  */
 Backdrop.tableDrag.prototype.showColumns = function () {
   // Show weight/parent cells and headers.
-  $('.tabledrag-hide', 'table.tabledrag-processed').css('display', '');
```

Comment:
is this change related to the label/machine update? I think it might have slipped in from our other tabledrag issue today.

Reply:
See my reply to the previous comment. This is to avoid "empty" `style` attributes left behind.